src/components/footer/footer.js

```diff
@@ -1,60 +1,81 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import Logo from "../shared/Logo/logo";
+import { NavLink } from "react-router-dom";
+import { IoMdSend } from "react-icons/io";
+import LogoWhite from "../../assets/icons/LogoWhite.png";
 import "./footer.scss";
-import constants from "../../constants/Footer";
+import FOOTER from "../../constants/Footer";
 
-const { SUBSCRIBE_US, SUBSCRIBE, CONTACT_US, PHONE_NUMBER } = constants;
+const {
+  SUBSCRIBE_US,
+  CONTACT_US,
+  PHONE_NUMBER,
+  PASTE_YOUR_EMAIL,
+  HOME,
+  CREATE_EVENT,
+  SEARCH_EVENT,
+  TERMS,
+  ABOUT,
+  STYLE_ICON_FACEBOOK,
+  STYLE_ICON_INSTAGRAM,
+  STYLE_ICON_YOUTUBE,
+  PATH_HOME,
+  PATH_EVENTS,
+  PATH_CREATE_EVENT,
+  PATH_TERMS,
+  PATH_ABOUT,
+  LINK_FACEBOOK,
+  LINK_INSTAGRAM,
+  LINK_YOUTUBE,
+} = FOOTER;
 
-const links = [
-  { id: 0, name: "Home", path: "/home" },
-  { id: 1, name: "Search Event", path: "/events" },
-  { id: 2, name: "Create Event", path: "" },
-  { id: 3, name: "Terms", path: "/terms" },
-  { id: 4, name: "About", path: "/about" },
+const LiNKS = [
+  { path: PATH_HOME, name: HOME },
+  { path: PATH_EVENTS, name: SEARCH_EVENT },
+  { path: PATH_CREATE_EVENT, name: CREATE_EVENT },
+  { path: PATH_TERMS, name: TERMS },
+  { path: PATH_ABOUT, name: ABOUT },
 ];
 
-const socialLinks = [
-  { id: 0, icon: "fab fa-facebook-f", link: "https://uk-ua.facebook.com" },
-  { id: 1, icon: "fab fa-instagram", link: "https://www.instagram.com" },
-  { id: 2, icon: "fab fa-youtube", link: "https://www.instagram.com" },
+const SOCIALLINKS = [
+  { icon: STYLE_ICON_FACEBOOK, link: LINK_FACEBOOK },
+  { icon: STYLE_ICON_INSTAGRAM, link: LINK_INSTAGRAM },
+  { icon: STYLE_ICON_YOUTUBE, link: LINK_YOUTUBE },
 ];
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_container">
+        <NavLink to={HOME}>
+          <img src={LogoWhite} alt="Logo" className="footer__logo" />
+        </NavLink>
+        <div className="footer__links">
+          {LiNKS.map(({ path, name }) => (
```

Comment:
That's was not done as I asked before, I mean regarding extracting this `LINKS.map` outside to component

Reply:
Done